hack/README.md

```diff
@@ -40,8 +40,10 @@ The standard workflow is performed inside a helper container to normalize the bu
     - `build-importer`: compile cdi-importer binary
     - No `build-cloner` target exists as the code is written in bash
 - `test`: execute all tests
-    - `test-unit`: execute all tests under `./pkg`
-    - `test-functional`: execute all tests under `./tests`
+    - `test-unit`: execute all tests under `./pkg/`
+    - `test-functional`: execute functional tests under `./tests/`. Reads from `TEST_ARGS` to pass through flags to tested code
+- `build-functest-image-init`: build the init container for the testing file server
```

Comment:
build-functest-image-init/http are not part of the Makefile, you can do make build build-functest-image-init build-functest-image-http instead.

Reply:
I still seem them here  https://github.com/kubevirt/containerized-data-importer/blob/master/Makefile#L46

Should they be removed?  They'd be useful to have rather than rebuilding all of CDI if changes have to be made to them.

Reply:
No you don't have to build all of cdi, you can just do 'make build build-functest-image-init' to build the image for init, similar for http.

Reply:
For a separate pr: I think target is a tad verbose. What do you think of deleting "BUILD" from the target name?

Reply:
That's probably the portion I'd be least keen to remove :) maybe `build-server-init`

Reply:
:smile:  Well, the reason I thought of removing _build_ is if the cmd is `make build build-functest-image-init` then "build" is repeated (this is what I thought I read in another comment). If the cmd is `make build-functest-image-init` then I take back my comment!